recipes/datma/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#chmod +x datma
+
+mkdir -p $PREFIX/bin/
+chmod +x *py
+cp *py $PREFIX/bin/
+chmod +x *sh
+cp *sh $PREFIX/bin/
+
+echo "export datmaPATH=$PREFIX/bin/" >> ~/.basrc
```

Comment:
This won't do anything. If datamaPath needs to be set (I suspect it doesn't) then you'll have to add   it in a `post-link.sh` script.